packages/react-components/src/App.jsx

```diff
@@ -1,7 +1,22 @@
+import { useEffect, useState } from 'react';
 import './App.css'
 import MIMap from './components/MIMap/MIMap'
 
 function App() {
+
+    const [mapsIndoorsInstance, setMapsIndoorsInstance] = useState();
+
+    useEffect(() => {
+        if (mapsIndoorsInstance) {
+            // For demo purpose: setup MapsIndoors click event listener that sets clicked location to be invisible.
+            mapsIndoorsInstance.on('click', (location) => {
+                const displayRule = mapsIndoorsInstance.getDisplayRule(location);
+                displayRule.visible = false;
+                mapsIndoorsInstance.setDisplayRule(location.id, displayRule);
+            });
```

Comment:
What was the use case for setting the clicked location to be invisible?

Reply:
Showcase. Demo. Nothing but that. It's not part of the Map component, but the test/demo app.